app/docs/azure-blob-storage-configuration.md

```
@@ -0,0 +1,206 @@
+
+Add the connection string to your environment variables:
+
+**Option A: Using `.env` file (Development)**
+
+1. Copy `config/.env.example` to `config/.env`
+2. Add the connection string:
+   ```bash
+   export AZURE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=mykey;EndpointSuffix=core.windows.net"
+   ```
+
+**Option B: Using System Environment Variables (Production)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)